### PR TITLE
nrf_wifi: Update RPU patch to rev#62892d9f23c

### PR DESCRIPTION
**Pull request by karun2796**
UMAC:
     Updated RPU patch version fropm 1.2.9.8 to 1.2.9.9.
     [SHEL-2394] : Updating Mismatched TX done statistics in SAP mode.
     [SHEL-2298] : Monitor mode support in UMAC.

LMAC:
     [SHEL-2397] : Proper rate flags updation in Rx event from LMAC.

[SHEL-2394]: https://nordicsemi.atlassian.net/browse/SHEL-2394?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SHEL-2298]: https://nordicsemi.atlassian.net/browse/SHEL-2298?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SHEL-2397]: https://nordicsemi.atlassian.net/browse/SHEL-2397?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_data_if.h [hunk 1]
```diff
@@ -186,7 +188,13 @@ struct nrf_wifi_rx_buff {
 	/** Header @ref nrf_wifi_umac_head */
 	struct nrf_wifi_umac_head umac_head;
 	/** Rx packet type. see &enum nrf_wifi_rx_pkt_type */
-	signed int rx_pkt_type;
+	signed short rx_pkt_type;
+	/** refer see &enum rpu_tput_mode */
```
Comment (sachinthegreen) [inline nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_data_if.h:192]: /** Refer @enum rpu_tput_mode */

File: nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_data_if.h [hunk 2]
```diff
@@ -186,7 +188,13 @@ struct nrf_wifi_rx_buff {
...
+	unsigned char rate_flags;
+	/** rate: legacy rates: 1,2,55,11,6,9,12,18,24,36,48,54
+	 *		  11N VHT HE  : MCS index 0 to 7.
```
Comment (sachinthegreen) [inline nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_data_if.h:195]:
```
/** Rate: Legacy : 1, 2, 55, 6, 9, 11, 12, 18, 24, 36, 48, 54
  *       11N VHT HE  : MCS index 0 to 7.
```

**Review by sachinthegreen — CHANGES_REQUESTED** [review]
Please raise manifest PRs in :

https://github.com/nrfconnect/sdk-nrf and,
https://github.com/NordicPlayground/nrf70-linux-driver

File: nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_data_if.h (hunk 2 above)
Comment (karun2796) [reply nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_data_if.h]: Taken Care

File: nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_data_if.h (hunk 1 above)
Comment (karun2796) [reply nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_data_if.h]: Taken care

**Review by rajb9 — APPROVED** [review]

**Review by udaynordic — APPROVED** [review]

**Comment by karun2796** [comment]
> Please raise manifest PRs in :
> 
> https://github.com/nrfconnect/sdk-nrf and, https://github.com/NordicPlayground/nrf70-linux-driver

Raised

**Review by sachinthegreen — APPROVED** [review]

**Pull request merged**